src/main/jbake/content/documentation/development/feature-model.md

```diff
@@ -11,4 +11,74 @@ standard. It has been created in Sling as that's its primary audience for now, b
 evolves.
 
 See [https://github.com/apache/sling-org-apache-sling-feature](https://github.com/apache/sling-org-apache-sling-feature) for more information
-about the Feature Model.
\ No newline at end of file
+about the Feature Model.
+
```

Comment:
I think it would be good to shorten this a little bit and simply reference to the documentation in the git repo for the feature model - to reduce duplication

Reply:
Great point. Duplication removed.